<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComposeDialog</class>
 <widget class="QDialog" name="ComposeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {border:1px solid gray;}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" alignment="Qt::AlignRight">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>SUBJECT:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3" colspan="2">
    <widget class="QPushButton" name="postButton">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>Post to Outbox</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="3" alignment="Qt::AlignRight">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>FROM:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" alignment="Qt::AlignRight">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>TO:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QComboBox" name="toComboBox">
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="4">
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" alignment="Qt::AlignRight|Qt::AlignTop">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>BODY:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QComboBox" name="fromComboBox">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="discardButton">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>Discard</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="4">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item row="4" column="0" colspan="2" alignment="Qt::AlignHCenter">
    <widget class="QCheckBox" name="p2pCheckBox">
     <property name="text">
      <string>P2P Message</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="ics213TemplateButton">
     <property name="text">
      <string>ICS-213 Template</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>toComboBox</tabstop>
  <tabstop>fromComboBox</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>plainTextEdit</tabstop>
  <tabstop>p2pCheckBox</tabstop>
  <tabstop>discardButton</tabstop>
  <tabstop>postButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
